packages/ibm-products/src/components/ConditionBuilder/ConditionGroupBuilder/ConditionGroupBuilder.js

```diff
@@ -31,7 +31,7 @@ const ConditionGroupBuilder = ({
       });
       handleFocusOnClose(e);
     } else {
-      onRemove();
+      onRemove(e);
```

Comment:
for consistency sake lets stick to `evt` for events

Reply:
Hey,
Modified as advised